perf/replicache.ts

```diff
@@ -167,30 +167,35 @@ export function benchmarkWriteSubRead(opts: {
   const numKeys = keysPerSub * numSubsTotal;
   const cacheSizeMB = (numKeys * valueSize) / 1024 / 1024;
   const kbReadPerSub = (keysWatchedPerSub * valueSize) / 1024;
-  const key = (k: number) => `key${k}`;
-
-  const data = jsonArrayTestData(numKeys, valueSize);
+  const makeKey = (index: number) => `key${index}`;
 
   return {
     name: `${
       opts.useMemstore ? '[MemStore] ' : ''
     }writeSubRead ${cacheSizeMB}MB total, ${numSubsTotal} subs total, ${numSubsDirty} subs dirty, ${kbReadPerSub}kb read per sub`,
     group: 'replicache',
     async run(bencher: Bencher) {
+      const keys = Array.from({length: numKeys}, (_, index) => makeKey(index));
+      const sortedKeys = keys.sort();
+      const data: Map<string, TestDataObject> = new Map(
```

Comment:
Moving data inside here was important
1. where it was data was already created during benchmark creation (execution of benchmarks function), blowing up memory usage before any benchmarks are run, and was never collected thus limiting how big we could make data before crashing the tab
2. before multiple test runs were mutating a shared data array